src/aleph/http/http2.clj

```diff
@@ -822,6 +822,8 @@
 
      :protocol              "HTTP/2.0"
 
+     ;; These keys are internal to Aleph and should not be relied on
+     :aleph/channel         ch
      :aleph/writable?       writable?
      :aleph/h2-exception    h2-exception
```

Comment:
If we add a new `:aleph.internal/` prefix, it should definitely apply to `channel`. Not sure about`writable?` and `h2-exception`. My gut says it shouldn't, but I can see scenarios where users might want to know about them in their handlers.

Reply:
Right, the good thing is that people may still use them if they so desire - it's just at their own peril then :smile: I would also make these internal since they are pretty low-level what with them being `AtomicBoolean`. Seems reasonable!

Reply:
> would also make these internal 

Just so we're on the same page, you mean we leave the prefix `:aleph/`, and keep the comment saying they're internal?

We can't change the `request-arrived` and `keep-alive?` prefixes.

Reply:
> Just so we're on the same page, you mean we leave the prefix :aleph/, and keep the comment saying they're internal?

Ah no I actually meant to change them to `:aleph.internal/`. But having just started doing that, I noticed we have a bunch of other `:aleph/` keywords which are probably also only used internally (e.g. `:aleph/omitted` or `:aleph/save-request-message` - haven't checked them all in detail, though). I suggest we leave that can of worms closed for now after all and punt this to a separate issue for future improvement where we can think this through more thoroughly. Let's keep using `:aleph/` throughout for the time being here and only move the comment so that it doesn't cover `:aleph/request-arrived` and `:aleph/keep-alive?`. Agree?

Reply:
👍

Reply:
Alright, pushed a commit to that effect and created https://github.com/clj-commons/aleph/issues/700 for future reconsideration.